docs/Manuals/Server/options.rst

```diff
@@ -0,0 +1,1176 @@
+
+  :strong:`Description`: :term:`AI` trait distribution method. Possible values:
+
+    * ``FIXED``: Fixed
+    * ``EVEN``: Even
+
+``turnblock``
+  :strong:`Value (Min, Max)`: enabled
+
+  :strong:`Description`: Turn-blocking game play mode. If this is turned on, the game turn is not advanced
+  until all players have finished their turn, including disconnected players.
+
+``unitwaittime``
+  :strong:`Value (Min, Max)`: 0 (0, 8639999)
+
+  :strong:`Description`: Minimum time between unit actions over turn change. This setting gives the minimum
+  amount of time in seconds between unit moves and other significant actions (such as building cities) after a
+  turn change occurs. For example, if this setting is set to 20 and a unit moves 5 seconds before the turn
+  change, it will not be able to move or act in the next turn for at least 15 seconds. This value is limited
+  to a maximum value of two-thirds of ``timeout``. ``unitwaittime`` (aka UWT) is a tool to help reduce
+  :term:`RTS` around turn change (:term:`TC`) in Longturn games.
+
+``unitwaittime_extended``
+  :strong:`Value (Min, Max)`: disabled
+
+  :strong:`Description`: ``unitwaittime`` also applies to newly-built and captured/bribed units. If set,
+  newly-built units are subject to ``unitwaittime`` so that the moment the city production was last touched
+  counts as their last *action*. Also, getting captured/bribed counts as action for the victim.
+
+``unitwaittime_style``
+  :strong:`Value (Min, Max)`: empty value/not set
+
+  :strong:`Description`: This setting affects ``unitwaittime`` and effectively unused as it only has one option
+  to set:
+
+    * Activities (``ACTIVITIES``): Units moved less than ``unitwaittime`` seconds from turn change will not
+      complete activities such as pillaging and building roads during turn change, but during the next turn
+      when their wait expires.
+
+``unreachableprotects``
+  :strong:`Value (Min, Max)`: enabled
+
+  :strong:`Description`: Does unreachable unit protect reachable ones. This option controls whether tiles with
+  both unreachable and reachable units can be attacked. If disabled, any tile with reachable units can be
+  attacked. If enabled, tiles with an unreachable unit in them cannot be attacked. Some units in some rulesets
+  may override this, never protecting reachable units on their tile.
+
+``victories``
+  :strong:`Value (Min, Max)`: Spacerace (``SPACERACE``) and Allied victory (``ALLIED``)
+
+  :strong:`Description`: What kinds of victories are possible for the game. This setting controls how a game
+  can be won. One can always win by conquering the entire planet, but other victory conditions can be enabled
+  or disabled:
+
+    * Spacerace (``SPACERACE``): Spaceship is built and travels to Alpha Centauri.
+    * Allied (``ALLIED``): After defeating enemies, all remaining players are allied.
+    * Culture (``CULTURE``): Player meets ruleset defined cultural domination criteria.
```

Comment:
The option values seem pretty self-explanatory, I don't think the short value descriptions are needed here
```suggestion
    * ``SPACERACE``: Spaceship is built and travels to Alpha Centauri.
    * ``ALLIED``: After defeating enemies, all remaining players are allied.
    * ``CULTURE``: Player meets ruleset defined cultural domination criteria.
```